app/javascript/src/april-fools.js

```diff
@@ -0,0 +1,27 @@
+import FetchRails from "./fetch-rails"
+
+export function destroy() {
+  const elements = document.querySelectorAll("[data-role~='april-fools']")
+
+  elements.forEach((element) => element.parentNode.remove())
+}
+
+export function bind() {
```

Comment:
You're not really binding anything, so techncailly it could just be `document.addEventListener("turbolinks:load", function() {`, but I'm not too concerned about it.

Reply:
I did this because while no event gets bound, it is still necessary to ensure that this happens before `dismissParent` fires so that the close button is bound. I will change the function name to avoid confusion though.

Reply:
Ah fair enough!